README.md

```diff
@@ -65,50 +99,83 @@ objdump syntax -> jasm/golbolt syntax
 
-So the example would match a command like `nopw 0x0(%rax,%rbx,1)` turning it to `[%rax+%rbx*1+0x0]`
+So the example would match a command like `nopw 0x0(%rax,%rax,1)` turning it to `[%rax+%rax*1+0x0]`
 
 ## Special registry capture groups
 
 The x86 registers are the following:
 
-![x86 registers](images/x86_registers.png)
+![x86 registers](x86_registers.png)
 
 we support the following convention for using special register captures.
 
 Use the capture group special names for capturing the registers:
 
-- `&genreg`
+* `&genreg`
 
-- `&indreg`
+* `&indreg`
 
-- `&stackreg`
+* `&stackreg`
 
-- `&basereg`
+* `&basereg`
 
 And these suffix for accessing the specific parts of the registers:
 
-- `.64`
+* `.64`
 
-- `.32`
+* `.32`
 
-- `.16`
+* `.16`
 
-- `.8H`
+* `.8H`
 
-- `.8L`
+* `.8L`
 
 So for example:
 
-- if you want to capture the `rax` register you can use `&genreg.64` and it will capture the `rax` register.
-- if you want to capture the `eax` register you can use `&genreg.32` and it will capture the `eax` register.
-- if you want to capture the `ax` register you can use `&genreg.16` and it will capture the `ax` register.
-- if you want to capture the `ah` register you can use `&genreg.8H` and it will capture the `ah` register.
-- if you want to capture the `al` register you can use `&genreg.8L` and it will capture the `al` register.
+* if you want to capture the `rax` register you can use `&genreg.64` and it will capture the `rax` register.
+* if you want to capture the `eax` register you can use `&genreg.32` and it will capture the `eax` register.
+* if you want to capture the `ax` register you can use `&genreg.16` and it will capture the `ax` register.
+* if you want to capture the `ah` register you can use `&genreg.8H` and it will capture the `ah` register.
+* if you want to capture the `al` register you can use `&genreg.8L` and it will capture the `al` register.
+
+## Configuration
+
+Global configuration for a JASM rule can be specified in the configuration section. The current options for configuration are:
+
+* style: Specifies the assembly style. Possible values include att for AT&T syntax.
+* mnemonics-full-match: A boolean value indicating whether the mnemonics should be matched fully.
+* operands-full-match: A boolean value indicating whether the operands should be matched fully.
+* sections: Specifies the sections to disassemble given a binary.
+Example:
+
+```yaml
+config:
+  style: att
+  mnemonics-full-match: true
+  operands-full-match: true
+  sections:
+    - ".plt"
+    - ".plt.got"
+```
+
+## Troubleshooting
+
+### Cannot install profiler-measure
+
+If poetry is failing to install this dependency, it may be because it cannot access gitlab credentials correctly.
+Create a `.netrc` file under your home directory with the following content:
+
+```
+machine gitlab.com login <eclypsium_email> password <gitlab_user_token>
+```
+
+This should allow poetry to access gitlab repositories and look for this package.
```

Comment:
In fact, I think we can just remove this whole section, as this plugin is not being used anymore.